### DISCUSSION TITLE
gh copilot CLI not working when using SSH authentication via 1Password

### DISCUSSION THREAD

**Discussion by fcamineiro** [github/gh-copilot#33] [Q&A]
I'm using 1Password to manage my SSH key authentication. It works fine with Git/GitHub for version control, but it returns an error message when using it with gh copilot CLI.

Example:
$ gh copilot explain "date -v +90d"

✗ Error: No valid OAuth token detected

To get started with GitHub Copilot in the CLI, please run: gh auth login --web -h github.com to authenticate via web browser.

$ gh auth status
github.com
  ✓ Logged in to github.com account _username_ (GITHUB_TOKEN)
  - Active account: true
  - Git operations protocol: https
  - Token: ghp_************************************
  - Token scopes: 'admin:org', 'repo'

  ✓ Logged in to github.com account _username_ (keyring)
  - Active account: false
  - Git operations protocol: https
  - Token: gho_************************************
  - Token scopes: 'gist', 'read:org', 'repo', 'workflow'

**Comment by kensternberg-authentik** [comment]
I am experiencing the same issue.  SSH or HTTPS, neither works:


```
$ gh auth status
github.com
  ✓ Logged in to github.com account kensternberg-authentik (GITHUB_TOKEN)
  - Active account: true
  - Git operations protocol: https
  - Token: [REDACTED]
  - Token scopes: 'audit_log', 'codespace', 'delete_repo', 'gist', 'notifications', 'project', 'read:org', 'read:packages', 'read:public_key', 'read:repo_hook', 'read:user', 'repo', 'user:email', 'user:follow', 'workflow', 'write:discussion'

  ✓ Logged in to github.com account kensternberg-authentik (keyring)
  - Active account: false
  - Git operations protocol: https
  - Token: [REDACTED]
  -  Token scopes: 'gist', 'read:org', 'repo', 'workflow'
```
